src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
+  padding: 32px 16px 16px;
+  height: 406px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(120%);
+    color: #34568b;
```

Comment:
Changing the text color on hover can affect readability. Ensure that the color contrast still meets accessibility standards.